.circleci/config.yml

```diff
@@ -309,13 +312,13 @@ commands:
                   BASE_PATH=/tmp/hermes/hermes-runtime-darwin/
                   if [ ! -d $BASE_PATH ]; then
                     echo "Hermes tarball base path not present ($BASE_PATH). Build it from source."
-                    return
+                    exit 1
```

Comment:
this was an error: we can't return in the script, we have to exit from the script earlier.